client/app/models/disposition.js

```diff
@@ -91,12 +91,16 @@ export default class DispositionModel extends Model {
   // e.g. 'Favorable', 'Conditional Favorable', 'Unfavorable', 'Conditional Unfavorable',
   // 'No Objection', 'Waiver of Recommendation', N/A is defualt
 
+
+  // TODO: These are duplicates??? Are they numbers or strings?
```

Comment:
what do you mean? one is borough **president*** the other is borough **board **

Reply:
> what do you mean? one is borough **president*** the other is borough **board **

did you look at the other attrs on the model? 

<img width="680" alt="image" src="https://user-images.githubusercontent.com/5004319/120016496-af3c9880-bfb2-11eb-86c1-f1ce5a39b41f.png">